tests/unit/test_canonical_vibe_entry_launcher.py

```
@@ -208,6 +259,216 @@ def fake_invoke_runtime(**kwargs: object) -> dict[str, object]:
+    assert prompt == "execute plan phase-cleanup"
+
+
+def test_resolve_effective_prompt_ignores_bounded_preferred_summary_without_explicit_allow(
+    tmp_path: Path,
+) -> None:
+    _write_bounded_return_summary(
+        tmp_path,
+        run_id="prior-bounded-run",
+        terminal_stage="xl_plan",
+        allowed_followup_entry_ids=["vibe", "vibe-do"],
+        reentry_token="token-123",
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "reentry_token"

(S106)